app/src/main/res/layout/item_egra_oral_vocabulary.xml

```diff
@@ -1,22 +1,36 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout
+<androidx.constraintlayout.widget.ConstraintLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
     android:id="@+id/egraOralVocabularyContainer"
     android:layout_width="wrap_content"
     android:layout_height="wrap_content"
-    android:padding="8dp"
-    android:gravity="center">
+    android:maxHeight="@dimen/spaceship_height"
+    tools:layout_width="@dimen/spaceship_width"
```

Comment:
Should `tools:layout_width` match the value of `android:layout_width` (for better consistency between the layout in the Android Studio preview and when running on a physical device?

Reply:
`item_*` layout doesn't have a defined size, its size is determined by `fragment_home_screen`. With `tools:layout_width="wrap_content"` images aren't visible in preview